<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cadastrarCliente</class>
 <widget class="QWidget" name="cadastrarCliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastrar Clientes  - [GeriFÃ¡cil] </string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>443</x>
     <y>100</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cidade:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>443</x>
     <y>60</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="in_nome">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>60</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_cidade">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>100</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="b_voltar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Voltar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="in_telefone">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>140</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>443</x>
     <y>140</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Telefone:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_limpar">
   <property name="geometry">
    <rect>
     <x>593</x>
     <y>200</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Limpar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_cadastrar">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>200</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Cadastrar</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>in_nome</tabstop>
  <tabstop>cb_cidade</tabstop>
  <tabstop>in_telefone</tabstop>
  <tabstop>b_cadastrar</tabstop>
  <tabstop>b_limpar</tabstop>
  <tabstop>b_voltar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
